### DISCUSSION TITLE
Setting up additional client causing settings to reset

### DISCUSSION THREAD

**Discussion by sytone** [remotely-save/remotely-save#115] [Q&A] [answered]
What would be the best process for setting up an additional client to sync to? 

I am using OneDrive and if I add another device I go through and install the plugin, auth and sync. I also tick replicate config and other folder so all my settings roam. However as the client makes a set of files when you make a new vault, they are technically newer and will in the sync logic. So I then go back and reenable plugins, settings etc. Or just hit OneDrive and copy the older version of the files over as it has history (yay).

Is there a better approach than this?

**Comment by fyears** [comment, marked as answer]
If you are talking about syncing the plugin settings or remotely-save by using remotely-save itself, I am sorry you have to go through the trouble.

If you are talking about syncing other plugin settings by using remotely-save. Then maybe it works: Sync the setting from device1. Then, when you make a new vault on device 2, you can avoid installing plugins on device 2 firstly, and just install remotely-save, and sync from the already uploaded file.

**Reply by uroybd** [reply]
I'm doing exactly this. However, facing a problem. The plugins keep appearing even if I delete them and sync the vault. Is it a bug, or am I confusing something?

**Reply by fyears** [reply]
don't sync the plugin folder
